### PR TITLE
Replace jQuery.timeago

### PR DESCRIPTION
**Pull request by shaneog**
Replace the heavy jQuery.timeago plugin with a smaller library.

- [x] remove jQuery.timeago
- [x] add [timeago.js](https://github.com/hustcc/timeago.js)
- ~~update with new translations ([PR open](https://github.com/hustcc/timeago.js/pull/5) on source library)~~
- [x] use `timeago.js` build with all locales included - waiting on [this issue](https://github.com/hustcc/timeago.js/issues/23)
- [x] set JS locale based on Rails locale

The list of included locales can be found [here](https://github.com/hustcc/timeago.js/tree/master/locales).

### CODE REVIEW

**Comment by mperham** [comment]
I had assumed the JS locale is provided by the browser on the client-side?

**Comment by hustcc** [comment]
Use `register` to load new locales, then you can use the locales translation~

**Comment by shaneog** [comment]
@mperham I could use the [navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language) API but I figured that it would be better to align fully with Rails/web app.

**Comment by mperham** [comment]
Good call.  Being consistent is more important than being right.

File: web/views/layout.erb [hunk 1]
```diff
@@ -29,5 +28,10 @@
     </div>
     <%= erb :_footer %>
     <%= erb :_poll_js %>
+    <script type="text/javascript">
```
Comment (connorshea) [inline web/views/layout.erb]: Please don't use inline scripts, this'd make enforcing Content Security Policies a lot more difficult.

Comment (shaneog) [inline web/views/layout.erb]: Do you have any suggestions as to how I can pass the value to JavaScript without using an inline script?

Comment (connorshea) [inline web/views/layout.erb]: Is there a reason why adding this exact code to the application.js (or equivalent) doesn't work?

Comment (shaneog) [inline web/views/layout.erb]: We pass a variable from Rails to the JavaScript app to load the correct locale (`<%= locale %>`).
I don't see how we can have the JS app use the same locale as Rails without passing this through.

Comment (shaneog) [inline web/views/layout.erb]: Also, it's worth noting that [`_poll_js`](https://github.com/mperham/sidekiq/blob/master/web/views/_poll_js.erb) already uses inline JavaScript.

Comment (mperham) [inline web/views/layout.erb]: You could set an attribute on <body> and then have the JS use that attribute.

Comment (connorshea) [inline web/views/layout.erb]: Could you just add a data attribute to the body element and get the locale from that?

e.g. `<body class="admin" data-locale="<%= locale %>">` and then

```javascript
var locale = $('body').data('locale');
window.addEventListener('load', function() {
  timeago().setLocale(locale);
});
```

Comment (shaneog) [inline web/views/layout.erb]: Perfect - I'll do that, thanks!

Comment (mperham) [inline web/views/layout.erb]: If someone wants to send a PR changing the poll_js logic in the same manner, I'd love it.

Comment (connorshea) [inline web/views/layout.erb]: Can I open an issue to make sure it doesn't get forgotten in the mean time? :)

Comment (mperham) [inline web/views/layout.erb]: Absolutely, that's what it's there for!

**Comment by shaneog** [comment]
OK, this PR is ready for review now. Sorry it took so long!

**Comment by mperham** [comment]
I don't like the "live update" at all.  Can we turn it off?  It will lead to bug reports: "Next retry: 10 seconds ago".

**Comment by mperham** [comment]
https://github.com/hustcc/timeago.js/pull/73

**Comment by shaneog** [comment]
Absolutely. PR incoming

**Comment by mperham** [comment]
Just did it.

**Pull request merged**